src/Components/Search/NewSearch/Mobile/OverlayBase.tsx

```diff
@@ -11,16 +11,19 @@ export const OverlayBase: FC<OverlayBaseProps> = ({ children, onClose }) => {
       dialogProps={{
         width: "100%",
         height: "100%",
-        background: "black",
+        background: "black100",
         justifyContent: "center",
-        backgroundColor: "white",
+        backgroundColor: "white100",
```

Comment:
❓

Reply:
Sorry, I misunderstood you 😅

Reply:
> Sorry, just wondering why that would be changed in a separate PR

I've started working on a feature, in which I completely re-implemented `OverlayBase`. I just want to avoid merge conflicts